### ISSUE TITLE
How to do customized edge feature normalization?

### ISSUE THREAD

**Issue by JinYang88** [dmlc/dgl#2602]
## ❓ Questions and Help

I would like to do edge feature normalization by grouping edges for src or dst nodes, which could be done by using this API https://docs.dgl.ai/en/0.4.x/generated/dgl.DGLGraph.group_apply_edges.html.

But this API is removed after 0.5.x, is there any alternative API to use? Thanks!

**Comment by yzh119** [comment]
Hi @JinYang88 may I ask you how do you normalized these features?

**Comment by JinYang88** [comment]
@yzh119 
For example, simply divide the weight by the sum of all the weights of incoming edges.

**Comment by yzh119** [comment]
Use the combination of `copy_e_sum` and `e_div_v` would work.
The first one aggregates (by sum) edge feature on destination nodes and the second one divides edge features by destination node features.

```
import dgl
import dgl.ops as op
import torch
g = dgl.rand_grpah(30, 100)
e = torch.rand(100, 3)
z = ops.copy_e_sum(g, e)
e_norm = ops.e_div_v(g, e, z)
```
